pages/common/duckdb.md

```diff
@@ -0,0 +1,32 @@
+# duckdb
+
+> Command-line client for DuckDB, an in-process analytical SQL engine.
+> More information: <https://duckdb.org>.
+
+- Start an interactive shell with a transient in-memory database:
+
+`duckdb`
+
+- Start an interactive shell on a database file. If the file does not exist, a new database is created:
+
+`duckdb {{path/to/dbfile.[db|duckdb]}}`
+
+- Load CSV or Parquet file to a table and quit:
+
+`duckdb {{path/to/dbfile.[db|duckdb]}} "{{CREATE OR REPLACE TABLE tbl AS FROM 'data_source.[csv|parquet]'}}"`
```

Comment:
Missing the command flag?

Reply:
Passing a command without `-cmd` / `-c` / `-s`: 

```console
$ duckdb dbfile.duckdb "SELECT 42 AS x";
┌───────┐
│   x   │
│ int32 │
├───────┤
│    42 │
└───────┘
```

Reply:
That said, the positional version can be confusing as `duckdb "SELECT 42 AS x"` will create a database file named `SELECT 42 AS x`. So I added the `-c` switch before the query.